src/features/ref.js

```diff
@@ -3,6 +3,5 @@ import styled from "styled-components";
 import { isForwardRef } from "react-is";
 
 const StyledComponent = styled.div``;
-const ref = isForwardRef(<StyledComponent />);
 
-export default ref;
```

Comment:
I don't know @plumbblake's opinion on this, but after working with him for a little bit I would think he'd prefer not to break the current stuff and add this feature as a named export instead (or a default export in another file, but that sounds like an extra file for little benefit).

Reply:
@robertwbradford I'm willing to make this change.  I think it it is a great improvement.  Could you update the test to reflect this change so that it will pass.  I'm also wondering if this would be possible to do for createGlobalStyle as well.  I wanted to keep the apis consistent so that they were always exporting booleans, but this changes them to have some exporting booleans and some exporting an object.

Reply:
Actually after giving this further thought. I'm not sure I want to do this. It adds opinion that I'm not sure I want to make.  I didn't want this library to assume what you wanted to do with the boolean check.  I'm teetering back and forth at the moment.

Reply:
If the export is boolean, should it be called "isRef"? Then you could have your generic boolean and the more helpful "getRefProp" if you wanted.

Reply:
Okay,  I'm going to add this feature, but make it a separate getRefProp utility.